zerver/lib/export.py

```diff
@@ -1311,8 +1312,10 @@ def import_uploads_local(import_dir: Path, processing_avatars: bool=False) -> No
             # new server's avatar salt
             avatar_path = user_avatar_path_from_ids(record['user_profile_id'], record['realm_id'])
             file_path = os.path.join(settings.LOCAL_UPLOADS_DIR, "avatars", avatar_path)
+            is_image_original = False
             if record['s3_path'].endswith('.original'):
                 file_path += '.original'
+                is_image_original = True
```

Comment:
What's the story with this variable?

Reply:
`ensure_medium_avatar_image` for LocalUploadsBackend(https://github.com/zulip/zulip/blob/master/zerver/lib/upload.py#L477-L487) creates the medium image using `.original` one. Hence we make sure the check makes sure that we create a medium sized image only when we have the `.original` one.

Reply:
Yeah, I see.  This has an undefined variable bug when not processing avatars, and also initializes the LocalUploadsBackend class in a loop.  I cleaned that up and merged as 0f183981e67d41d017d88072a7024e1f81236121.  Can you test this against Slack (and the normal import) and make sure it still works?

Reply:
Ahh, I totally missed this, I should have initialised it in the records loop itself. Thanks for pointing it out, I'll be careful.